website/src/content/development/testing.mdx

```diff
@@ -30,3 +32,500 @@ To test React components, we recommend to use one of the following approaches:
   </Warning>
 
 For End-to-End tests you should use [Cypress](https://www.cypress.io/). Please check out the [app-kit cypress setup](https://github.com/commercetools/merchant-center-application-kit/tree/master/cypress) for more information.
+
+# Test utils for `<ApplicationShell>`
```

Comment:
This is basically the readme of the appshell test-utils. I moved it here and reworked that a bit to make it more readable.